### ISSUE TITLE
Kaltura Videos on Demo Page Won't Play or Seek

### ISSUE THREAD

**Issue by kangaree** [cookpete/react-player#1202]
#### Current Behavior
Kaltura Videos on the Demo Page do not play or seek when using the Play/Pause button or Seek range input. 

Perhaps this is related to
https://github.com/cookpete/react-player/pull/1082#issuecomment-759603500
?

Videos are playable if you click the Kaltura video or have the video muted before clicking Play (but you cannot Pause).

#### Expected Behavior
Kaltura Test A and Test B should be able to play or seek. 

#### Steps to Reproduce

1. Use 2.9.0. which just added Kaltura (https://cookpete.com/react-player/ is 2.6.2).
2. Run it locally:
```
git clone https://github.com/CookPete/react-player.git
cd react-player
npm install # or yarn
npm start
open http://localhost:300
```
3. Pick any of the Kaltura options- TestA or TestB.
4. Toggle the Play/Pause button
5. Click on a point in the Seek range after the beginning of the video 

#### Environment
- URL attempting to play: http://localhost:3000/
- Browser: Chrome Version 89.0.4389.90 (Official Build) (x86_64)
- Operating system: MacOS Catalina

#### Other Information
Provided demo videos:
https://cdnapisec.kaltura.com/p/2507381/sp/250738100/embedIframeJs/uiconf_id/44372392/partner_id/2507381?iframeembed=true&playerId=kaltura_player_1605622074&entry_id=1_jz404fbl
https://cdnapisec.kaltura.com/p/2507381/sp/250738100/embedIframeJs/uiconf_id/44372392/partner_id/2507381?iframeembed=true&playerId=kaltura_player_1605622336&entry_id=1_i1jmzcn3

If you add `allow='encrypted-media;autoplay'` to Kaltura.js, the video can be played but not paused.

**Comment by kangaree** [comment]
@jessp01 
Could you take a look into this if you have a moment? 

Can you still use Pause/Play and Seek in the demo app (src/demo/App.js) locally?

Perhaps something changed with the Kaltura media? I can't get Kaltura videos to Pause/Play or Seek using this branch https://github.com/jessp01/react-player/tree/support-kaltura.

I was excited to see Kaltura being added to react-player! Any help would be much appreciated!

**Comment by Yasamato** [comment]
A month later already, but I think my workaround is working (at least the demo). You may want to try it @kangaree at #1226

**Comment by cookpete** [comment]
Fixed in `2.10.0`
